src/ucp/core/ucp_request.c

```diff
@@ -317,8 +317,13 @@ int ucp_request_pending_add(ucp_request_t *req)
 static unsigned ucp_request_dt_invalidate_progress(void *arg)
 {
     ucp_request_t *req = arg;
+    ucp_ep_h ucp_ep    = req->send.ep;
 
     ucp_request_complete_send(req, req->status);
+
+    /* Now UCP endpoint could be destoyed if it was asked */
```

Comment:
`destroyed`, but I'd remove this comment, it is not really reflects what is done here (decrement a reference count hsich is clear from the name)

Reply:
removed